packages/stark-ui/src/modules/language-selector/components/language-selector.component.spec.ts

```diff
@@ -0,0 +1,148 @@
+/* tslint:disable:completed-docs */
+/* angular imports */
+import { Component, ViewChild } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { MatButtonToggleModule } from "@angular/material/button-toggle";
```

Comment:
The path of this import should also be included in this file: `packages/rollup.config.common-data.js` otherwise Rollup will display a warning for this import while building the Stark packages